src/start/RealmScreen.js

```diff
@@ -84,13 +84,20 @@ class RealmScreen extends React.Component {
       this.setState({ progress: false });
     }
   };
+  static contextTypes = {
+    styles: () => null,
+  };
 
   render() {
     const { styles } = this.context;
     const { progress, realm, error } = this.state;
+    const backgroundColor = StyleSheet.flatten(styles.background).backgroundColor;
 
     return (
       <Screen title="Welcome" keyboardAvoiding>
+        <ZulipStatusBar
```

Comment:
There is already `ZulipStatusBar ` in `Screen`. Can we do this directly [here](https://github.com/zulip/zulip-mobile/blob/master/src/common/Screen.js#L40)